src/logger/sampleRecord.c

```diff
@@ -54,6 +56,43 @@ void free_sample_buffer(struct sample *s)
         s->channel_samples = NULL;
 }
 
+bool get_sample_value_by_name(struct sample *s, const char * name, double *value)
+{
+    if (!s || !value) return false;
```

Comment:
What about name here?  If that is null should it do anything?

Reply:
I should catch that. I think STR_EQ would choke on a null string.

Reply:
fixed.